frontend/.gitignore

```diff
@@ -13,6 +13,7 @@
 
 # misc
 .DS_Store
+.env*.local
```

Comment:
Isn't this redundant due to the explicit listing below?

Reply:
Yes, this can be seen as redundant. I just wanted to make sure so. This can be removed.